wikipediapreview.php

```diff
@@ -63,6 +67,10 @@ function wikipediapreview_detect_deletion() {
 }
 
 function wikipediapreview_guten_enqueue() {
+	$enabled_post_types = apply_filters(
+		'wikipediapreview:enabled_post_types',
+		array('post', 'page')
+	);
 	if ( ! in_array( get_post_type(), array( 'post', 'page' ), true ) ) {
```

Comment:
Did you want to use the `$enabled_post_types` variable here?

Reply:
Yes! It should be used in the same way as line 23, my mistake :P